src/store/normalize-data.js

```diff
@@ -201,6 +204,12 @@ const normalizeData = (data) => {
     return state;
   }
 
+  // check the current parameters flag to determine if parameters are disabled
+  const flagState = getFlagsFromUrl();
+  if (flagState.parameters) {
+    state.nodeType.disabled.parameters = false;
+  }
```

Comment:
@studioswong This only gets flags from URL - it doesn't know about localStorage values or the initial default values. So if a flag is set to true by default, or true in localStorage, it will be ignored here.

Reply:
Thanks @richardwestenra ! That solution was done given an misunderstanding that the parameters flag was not stored in localstorage, hence I opted for a more straight forward approach! I see that you had implemented a new solution now, so I believe it was been resolved :)